pkg/logger/ecs_error_log.go

```diff
@@ -0,0 +1,201 @@
+package logger
+
+import (
+	"container/list"
+	"fmt"
+	"go.uber.org/zap/zapcore"
+	"log"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+)
+
+
+var UnixErrorLogPath = "/opt/aws/aws-otel-collector/logs/aws-otel-collector-error.log"
+
+var WindowsErrorLogPath = "C:\\ProgramData\\Amazon\\AWSOTelCollector\\Logs\\aws-otel-collector-error.log"
```

Comment:
Out of curosity, did we test it on windows?

Reply:
Its okay if it doesn't work on Windows yet. Only linux is required.

Reply:
Then maybe we should remove it for now. We will introduce them later when we will add them after testing.